docs/Maths.md

```diff
@@ -47,6 +47,76 @@ Result:
 
 ![](matplotlib.png)
 
+### Using Pandas ###
+Create a plot using [pandas.DataFrame.plot](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html)
+
+```python
+from fpdf import FPDF
```

Comment:
Could you auto-format this code with `black` code formatter please?
This would make the code style more consistent in the docs 🙂
If ever you don't manage to it, I'm OK to do it after the PR is merged

Reply:
Sure, I will check with black code formatter